app/services/analyze.py

```
@@ -325,15 +228,117 @@ async def _generate_documentation(self,
-            for chunk in response:
-                if chunk.choices[0].delta.content:
+            
+            async for chunk in stream:
+                delta = getattr(chunk.choices[0].delta, "content", None)
+                if delta:
+                    if delta in ("json", "```"):
+                        delta = ""
+                    yield delta
+                await asyncio.sleep(0)
+        
+        except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)